roles/openshift/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Common variables
+openshift_env: 'test'
+openshift_roles: [ 'master', 'node' ]
+openshift_bind_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
The main goal for the defaults was to simplify the POC/demo case, which is something that we get dinged for a lot with the current methods.

That said, I wouldn't be opposed to creating a wrapper role for the demo/simplistic case that sets the defaults and then calls the role without defaults.